references/classification/train_quantization.py

```diff
@@ -257,9 +257,17 @@ def get_args_parser(add_help=True):
     parser.add_argument("--clip-grad-norm", default=None, type=float, help="the maximum gradient norm (default None)")
     parser.add_argument("--weights", default=None, type=str, help="the weights enum name to load")
 
+    parser.add_argument("--backend", default="PIL", type=str.lower, help="PIL or tensor - case insensitive")
+    parser.add_argument("--use-v2", action="store_true", help="Use V2 transforms")
```

Comment:
This is the actual fix. These fields are accessed in 

https://github.com/pytorch/vision/blob/a63f98add5d67f85a5eac6ede8ca850d3b2bbfd3/references/classification/train.py#L147-L148

but since the `args` parameter comes from this script (and not from `train.py`), they need to be defined here as well.


Since the `--backend` parameter already existed in this file, I decided to break BC and rename it into `qbackend`.

Reply:
This needs to be reflected in the README, e.g.

https://github.com/pytorch/vision/blob/a63f98add5d67f85a5eac6ede8ca850d3b2bbfd3/references/classification/README.md?plain=1#L292